### ISSUE TITLE
Device group not controllable via speech

### ISSUE THREAD

**Issue by rhannink** [swttt/com.swttt.devicegroups#5]
Hello,

I created a group in the living with all the lights and call this group "schemerlampen". I would expect that I can turn them on by saying "switch on schemerlampen". But "schemerlampen" is not a know device.

**Comment by swttt** [comment]
This should work. Did Homey understand you correctly?

**Comment by rhannink** [comment]
Yep, the speech-input page shows: "homey: schakel de schemerlampen aan" but nothing happens.

**Comment by swttt** [comment]
Still didn't had any time to look into the issue. Will do asap

**Comment by swttt** [comment]
Really weird, can't reproduce.
isn't it conflicting with any other device named somehow the same?

What if you type in the speech-input page?

**Comment by rhannink** [comment]
Bas, wil try again this weekend.....
